tycho-artifactcomparator/src/main/java/org/eclipse/tycho/artifactcomparator/ArtifactComparator.java

```diff
@@ -19,22 +19,22 @@
 
 public interface ArtifactComparator {
 
-    public static class ComparisonData {
+    public static record ComparisonData(List<String> ignoredPattern, boolean writeDelta, boolean showDiffDetails) {
```

Comment:
Maybe this change can be a separate PR.

Reply:
Sure. Here we go: https://github.com/eclipse-tycho/tycho/pull/2610